utils/index.js

```diff
@@ -0,0 +1,5 @@
+const email = require('./email');
+
+module.exports = {
+  email,
```

Comment:
Will email be an object or a function?
If it will be a function then it is fine, otherwise it is a bug...

Reply:
i am exporting the function here also